File: Programmers/Java/Level 1/0803/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String []tmp = s.split("");
        String space = " ";
        int value=0;
        for(int i=0; i<tmp.length; i++){
            if(tmp[i].equals(space)){
                value = 0;
            }
            else{
                if(value%2 == 0)
                    tmp[i] = tmp[i].toUpperCase();
                
                else
                    tmp[i] = tmp[i].toLowerCase();
                value++;
            }
            answer += tmp[i];
        }
        return answer;
    }
}

// 참고

File: Programmers/Python/이상한 문자 만들기.py
def solution(s):
    answer = ''
    cnt = 0
    for i in s:
        if i == " ":
            answer += " "
            cnt = 0
        elif cnt % 2 == 0:
            answer += i.upper()
            cnt += 1
        else :
            answer += i.lower()
            cnt += 1
        
    return answer